_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -0,0 +1,114 @@
+
+Naturally, since this was such a massive change, several bugs also slipped through. But [@avently](https://github.com/avently), with the help of [@blackbox87](https://github.com/blackbox87), [@Stypox](https://github.com/Stypox) and [@TobiGr](https://github.com/TobiGr) fixed them rapidly as they were reported. 
+
+## The Notification is dead, long live the Notification!
+
+If you care about such things, you have likely noticed that NewPipe's notification looks quite different from most media player apps these days. It has a progress bar, buttons in odd places, and a permanent grey colour. This is because NewPipe uses an old API for notifications called RemoteView. This allowed for a lot of customisation, and that was good at the time it was in vogue. However, it has been deprecated and superseded by a more modern API.
+
+Enter the **Age of Mediastyle notification**:
+
+This is a special notification type which has been available for media apps to use since Android Lollipop. In Oreo, it gained the ability to be coloured to match the colour theme of the cover art of the music playing. And since Android 10, it has a seekbar built right into it! It is a powerful API that removes the need to use a whole bunch of custom code to achieve what we had so far. This will make it far easier to maintain and improve NewPipe's notification. Win for development! So let's get on with what is new and what has changed.
+
+### Android 9
+
+First of all, we say "sorry" to our users on Android 9 and below. The progress bar is gone. As we've said above, it will only be shown on Android 10 and above. We're a small team and can't maintain code for the old and new types of notifications simultaneously, so the old one has to go. And not just that; the old notification isn't even supported Android 11 onwards. Bye-bye RemoteView notifications. We shall miss you! :'(
+
+But you get a lot in exchange for that sacrifice!
+
+### New features
+
+You get five CUSTOMISABLE - you read it right! - **CUSTOMISABLE** action buttons to control playback. There is a new notification menu in the apps settings, and you can choose which button should perform which player action. Play/pause, forward/rewind, previous/next, shuffle, repeat, close, or even nothing!
+
+Then, you can choose which three of these five buttons show in the compact notification (remember that little arrow you use to expand/contract the notification?).
+
+For users on **Android 8+**: Colours! Warm colours. Cool colours. All the colours! As we've noted above, Mediastyle notifications adapt to the colours of the cover art, which looks just plain awesome.
+
+For users on **Android 10+**: Seekbar! The old notification had a progress bar, but that was just a visual indicator. Now you can actually seek from the notification itself. Marvel at the massive boost in convenience, and have fun seeking!
```

Comment:
Why does the word after the colon get capitalized here but not later in this post? It should be consistent

Reply:
No, it can be both. Basically, if a colon replaces a dot, you capitalize, if it replaces a comma, you dont.


https://www.grammarly.com/blog/capitalization-after-colons/

Reply:
Interesting… you should follow whatever is correct for British English (or at least, you're using e.g. "colour" in the rest of the article)

Reply:
Oh, that was me. India has British spellings instead of American. I have no preference regarding that. Either way would be fine for me.

Reply:
I guess resolved?